exercises/concept/guidos-gorgeous-lasagna/lasagna_test.py

```diff
@@ -1,11 +1,26 @@
 import unittest
 import pytest
-from lasagna import (
-                    EXPECTED_BAKE_TIME,
-                    bake_time_remaining,
-                    preparation_time_in_minutes,
-                    elapsed_time_in_minutes
-                    )
+import sys
+
+try:
+    from lasagna import (
+                        EXPECTED_BAKE_TIME,
+                        bake_time_remaining,
+                        preparation_time_in_minutes,
+                        elapsed_time_in_minutes
+                         )
+
+
+except ImportError as import_fail:
+    message = import_fail.args[0].split('(')[0]
+    function_name = import_fail.args[0].split()[3][:-1] + "()'"
+
+    if 'EXPECTED_BAKE_TIME' in message:
+        raise ImportError(f"We can't the find the constant 'EXPECTED_BAKE_TIME' in your 'lasagna.py'"
```

Comment:
`pylint` ought to catch this line for being an f-string without any formatting.

Reply:
It does, `pylint-testing.py:4:25: W1309: Using an f-string that does not have any interpolated variables (f-string-without-interpolation)`, but I do not think is a big issue. It is not in PEP8, which is the code-style we really do adhere to. I don't think it is an issue and I'll leave it up to @BethanyG to decide.